clang/lib/Sema/SemaSYCL.cpp

```diff
@@ -3477,9 +3481,8 @@ static const char *paramKind2Str(KernelParamKind K) {
     CASE(std_layout);
     CASE(sampler);
     CASE(pointer);
-  default:
-    return "<ERROR>";
   }
+    return "<ERROR>";
```

Comment:
Please fix the clang-format error about indentation here.    But otherwise, looks good.